main.go

```diff
@@ -49,7 +49,7 @@ func TransformExportJsonRecord(wh warehouse.Warehouse, rec map[string]interface{
 	return line, nil
 }
 
-func ProcessExportsSince(wh warehouse.Warehouse, since time.Time) (int, error) {
+func ProcessExportsSince(wh warehouse.Warehouse, since time.Time, processFunc func(warehouse.Warehouse, *fullstory.Client, []fullstory.ExportMeta) (int, error)) (int, error) {
```

Comment:
Nit: you may want to type alias the function signature to make it easier to reference.